pkg/xray/resource_xray_security_policy_test.go

```diff
@@ -628,7 +628,7 @@ func TestAccSecurityPolicy_vulnerabilityIdsConflictingAttributesFail(t *testing.
 func TestAccSecurityPolicy_vulnerabilityIdsLimitFail(t *testing.T) {
 	_, fqrn, resourceName := test.MkNames("policy-", "xray_security_policy")
 	testData := util.MergeMaps(testDataSecurity)
-	CVEString := generateListOfNames("CVE-2022-", 101)
+	CVEString := generateListOfNames("CVE-2022-", 102)
```

Comment:
This should be 101, right?

Reply:
102 in the function creates 101 CVEs, it should be 102 to fail. This is a negative test.

Reply:
That's counter-intuitive. I'd suggest changing the function so that when it is called with 101, 101 items are created.

Reply:
Ok, changed.